src/smartthings/amazon-echo-smartapp-v2.groovy

```diff
@@ -1016,44 +1027,106 @@ def String batteryStatusReminder(List devices = null) {
     return outputText
 }
 
+def armSHMCommandHandler() {
+    log.trace "armSHMCommandHandler()"
+    Map desiredSHMStatus = newSHMStatus('away')
+    String outputText = ""
+    // check the current state
+    Map beforeSHMStatus = querySHMStatus()
+    if (beforeSHMStatus.mode == SHMModeDef.AWAY) {
+        outputText = beforeSHMStatus.sayStatus("is already") + " "
+        outputText += "To change this to $desiredSHMStatus.stringStatus(), visit the Dashboard in your SmartThings app."
```

Comment:
I am not sure if Dashboard is a good reference, I would just say Smart Home Monitor